### ISSUE TITLE
Rails/FilePath - False positive on blob with slash

### ISSUE THREAD

**Issue by rnickmccandless** [rubocop-hq/rubocop#6896]
Code that returns a listing of sub directories, without files, within a Rails root raises an offense:

> Rails/FilePath: Please use Rails.root.join('path', 'to') instead.

## Expected behavior

When running `$ rubocop` on any ruby file within a Rails project:

    Dir[Rails.root.join('app', 'services', '{*/}')]

Should _not_ raise the "Rails/FilePath" offense for the blob slash that returns a sub directory listing. 

## Actual behavior

    Dir[Rails.root.join('app', 'services', '{*/}')]

Raises the "Rails/FilePath" offense due to the slash. 

## Steps to reproduce the problem

When running the `$ rubocop`, a "Rails/FilePath" offense is raised on any ruby file within a Rails project that has the following code: `Rails.root.join('<dir>', '<dir>', '{*/}')`. 

## RuboCop version
```
0.67.2 (using Parser 2.6.2.1, running on ruby 2.5.3 x86_64-darwin18)
```

**Comment by rnickmccandless** [comment]
An example I'm looking for without triggering the rubocop offense is to return a list of directories only (without the included files):

```ruby
Dir[Rails.root.join('app', 'models', '{*/}')]
# => ["/Users/rnickmccandless/Projects/ruby/my-project/app/models/users/", "/Users/rnickmccandless/Projects/ruby/my-project/app/models/concerns/"]
```

Currently, I have to disable the cop to get by:
```ruby
# rubocop:disable Rails/FilePath
model_dir_paths = Dir[Rails.root.join('app', 'models', '{*/}')]
# rubocop:enable Rails/FilePath
```

...this seems like a bug.
